src/Components/Actions/ActionsButton.js

```diff
@@ -1,6 +1,11 @@
 import React from 'react';
 import './ActionsButton.css';
+import { componentsTitles } from 'Texts/texts';
 
 export default function ActionButton() {
-  return <div id="actionsButton">Actions</div>;
+  return (
+    <div id="upperRightButtonsContainer">
```

Comment:
is this div necessary or in the right place?
also - change it's name to be more descriptive, and move it's css to the place it exists.